libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-roles.component.ts

```diff
@@ -113,4 +128,9 @@ export class DhAdminPermissionRolesComponent implements OnInit, OnChanges, OnDes
       this.getPermissionQuery?.refetch({ id });
     }
   }
+
+  translateHeader = (columnId: string): string => {
+    const baseKey = 'admin.userManagement.permissionDetail.tabs.userRoles.columns';
+    return this.trans.translate(`${baseKey}.${columnId}`);
+  };
```

Comment:
nitpick: unintentionally has sneaked in :)

Reply:
https://github.com/Energinet-DataHub/greenforce-frontend/pull/1589/commits/ffe5f38eb1d959573b35edf79f19a6be9d0c190c